.travis.yml

```diff
@@ -8,6 +8,11 @@ go:
   - 1.11.x
```

Comment:
You can remove this here since you've got it in the matrix now. Otherwise you'll get a dup job (possibly intentional?)
Seen here: https://travis-ci.org/metaverse/truss/builds/483689196

Can probably also remove tip and put it in the below matrix as well

Reply:
It was intentional since 1.11's primary dependency management is still vendor so I figured it would be best to test for both until 1.12 comes out and defaults to modules. If you'd like to only test for 1.11 modules that's totally cool with me though and I can move tip down there as well.